### PR TITLE
[WIP] Pre release changes

### PR DESCRIPTION
**Pull request by bravegrape**
Final changes before releasing the new documentation

### CODE REVIEW

File: source/get-started-with-vsp/index.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Get started with the Verify Service Provider
+
+The Verify Service Provider (VSP) allows you to send and receive messages from the GOV.UK Verify Hub. GOV.UK Verify uses [Security Assertion Markup Language (SAML)][saml] for secure message exchange. Using the VSP removes the need for your service to handle SAML by:
+
+- generating SAML requests to send to the GOV.UK Verify Hub
+- translating SAML responses from the GOV.UK Verify Hub into JSON
+
+## Set up the Verify Service Provider
+
+The first step in connecting to GOV.UK Verify is setting up the VSP locally. You must host the VSP on your own infrastructure and you can connect multiple services to one instance.
```
Comment (jonathanglassman) [inline source/get-started-with-vsp/index.html.md.erb]: Could change "setting up the VSP locally" to "to set up the VSP locally"

File: source/get-started-with-vsp/index.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+To start setting up the VSP locally, [download the latest release][vsp-release] from GitHub and configure it using the information in the `README.md` file.
+
+[Follow the VSP tutorial][tutorial] to make sure your service correctly uses the VSP to:
+
+- generate and send requests
+- handle all possible responses
+
+The tutorial uses a placeholder for GOV.UK Verify Hub so you can follow along using your local VSP setup.
```
Comment (jonathanglassman) [inline source/get-started-with-vsp/index.html.md.erb]: Could change to "The tutorial uses a placeholder for the GOV.UK Verify Hub so you can follow the tutorial using your local VSP setup."

Comment (bravegrape) [reply source/get-started-with-vsp/index.html.md.erb]: YES! That sounds much better 👍

File: source/get-started-with-vsp/index.html.md.erb (hunk 1 above)
Comment (bravegrape) [reply source/get-started-with-vsp/index.html.md.erb]: 👌

**Pull request merged**